turbohare/hare.go

```diff
@@ -64,13 +66,19 @@ func (h *SuperHare) Start(ctx context.Context) error {
 					if layerID.GetEpoch().IsGenesis() {
 						logger.With().Info("not sending blocks to mesh for genesis layer")
 						return
-					} else if layerBlocks, err := h.mesh.LayerBlockIds(layerID); err != nil {
-						logger.With().Error("error reading block ids for layer, not sending to mesh",
+					}
+					blocks, err := h.mesh.LayerBlockIds(layerID)
+					if err != nil {
+						logger.With().Warning("error reading block ids for layer, using empty set",
 							layerID,
 							log.Err(err))
-					} else {
-						h.mesh.HandleValidatedLayer(ctx, layerID, layerBlocks)
 					}
+					if len(blocks) == 0 {
```

Comment:
you can mark https://github.com/spacemeshos/go-spacemesh/issues/2795 resolved by this PR @dshulyak

Reply:
oh wait. this change is only for turbo hare...